src/main/java/uk/ac/ebi/biostudies/controller/Mining.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.ebi.biostudies.controller;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ebi.biostudies.service.TextMiningLinkUpdater;
+
+import static uk.ac.ebi.biostudies.api.util.Constants.JSON_UNICODE_MEDIA_TYPE;
+
+/**
+ * Temporary url to test text mining msg rcv
+ */
+
+@RestController
+@RequestMapping(value="/api/v1/mining")
```

Comment:
Following REST endpoint naming conventions, mining should probably be a noun. Link?

Reply:
mining is not noun?

Reply:
Sorry, I meant a more specific noun. How about minedlink or something similar. Let's discuss this in the meeting today.